src/Compute/Compute/Compute.csproj

```diff
@@ -13,6 +13,7 @@
 
   <ItemGroup>
     <PackageReference Include="AutoMapper" Version="6.2.2" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.7.0-preview.1" />
```

Comment:
@blueww Hi Wei Wei, do you have any concerns that Compute module uses storage track 2 SDK?

Reply:
Why need to use the storage track2 SDK?

Reply:
I went back to using older existing objects instead of BlobClient. No longer using the new track2 sdk.